user_guide_src/source/libraries/email.rst

```diff
@@ -113,35 +113,37 @@ sending email.
 =================== =================== ============================ =======================================================================
 Preference          Default Value       Options                      Description
 =================== =================== ============================ =======================================================================
-**userAgent**       CodeIgniter         None                         The "user agent".
-**protocol**        mail                ``mail``, ``sendmail``,      The mail sending protocol.
+**$fromEmail**                                                       The email address to be set in the "from" header.
+**$fromName**                                                        The name to be set in the "from" header. 
+**$userAgent**      CodeIgniter                                      The "user agent".
+**$protocol**       mail                ``mail``, ``sendmail``,      The mail sending protocol.
                                         or ``smtp``
-**mailPath**        /usr/sbin/sendmail  None                         The server path to Sendmail.
-**SMTPHost**        No Default          None                         SMTP Server Hostname.
-**SMTPUser**        No Default          None                         SMTP Username.
-**SMTPPass**        No Default          None                         SMTP Password.
-**SMTPPort**        25                  None                         SMTP Port. (If set to ``465``, TLS will be used for the connection
+**$mailPath**       /usr/sbin/sendmail                               The server path to Sendmail.
+**$SMTPHost**                                                        SMTP Server Hostname.
+**$SMTPUser**                                                        SMTP Username.
+**$SMTPPass**                                                        SMTP Password.
+**$SMTPPort**       25                                               SMTP Port. (If set to ``465``, TLS will be used for the connection
```

Comment:
Please remove all `$` marks in `Preference`.
Because these items may be keys in an array.